client/my-sites/themes/controller/index.node.js

```diff
@@ -42,27 +40,14 @@ export function fetchThemeDetailsData( context, next ) {
 	}
 
 	const themeSlug = context.params.slug;
-	const theme = themeDetailsCache.get( themeSlug );
+	const theme = getThemeDetails( context.store.getState(), themeSlug );
 
-	if ( theme ) {
+	if ( Object.keys( theme ).length ) {
```

Comment:
Since we now rely on the Redux store for caching (which is initialized on a per-request basis [here](https://github.com/Automattic/wp-calypso/blob/master/server/isomorphic-routing/index.js#L44)), I'm thinking that the cache probably doesn't even live longer than a request. Which kinda devaluates the whole thing. :-/ (BTW I think it wasn't quite working before either, since `if ( theme )` yields `true` for an empty object.)